homeassistant/components/knx/factory.py

```diff
@@ -212,6 +215,7 @@ def _create_sensor(knx_module: XKNX, config: ConfigType) -> XknxSensor:
         name=config[CONF_NAME],
         group_address_state=config[SensorSchema.CONF_STATE_ADDRESS],
         sync_state=config[SensorSchema.CONF_SYNC_STATE],
+        always_callback=config[SensorSchema.CONF_ALWAYS_CALLBACK],
```

Comment:
to have sensors fire a HA state change event on every telegram received. Eg. to have more accurate logs of slowly changing states

Reply:
If the incoming reading from a (binary) sensor comes directly from the device and not a cache, we should return `True` from the `force_update` entity property.

Reply:
In what case would we not want to always tell Home Assistant there is updated data ? Seems like a conflation of concerns. The API should get the data to HA as-is, and the entity should decide if they want to drop this update or not.

Reply:
The logic right now is: If two payloads are identical we do not trigger an update callback. With `always_callback` being `True` we do.

I'll add the `force_update`, that sounds like a nice addition!

Reply:
Property added.